### DISCUSSION TITLE
Bitcoincore Daemon "mainnet" (bitcoin) setup and next steps

### DISCUSSION THREAD

**Discussion by w3evolutions** [L2-Technology/sensei#152] [Q&A]
I'm not sure I understand how all this works, so this may not be the right question(s) to ask, but I will do my best!

I am running btcoincore daemon.

I have connected RPC from the cargo cmd:
```dos
> cargo run --bin senseid -- --bitcoind-rpc-host=127.0.0.1 --bitcoind-rpc-port=8455 --bitcoind-rpc-username=X --bitcoind-rpc-password=X --database-url=sensei.db
```
```
Finished dev [unoptimized + debuginfo] target(s) in 0.46s
Running \`target\debug\senseid.exe --bitcoind-rpc-host=127.0.0.1 --bitcoind-rpc-port=8455 --bitcoind-rpc-username=X --bitcoind-rpc-password=X --database-url=sensei.db\`
manage your sensei node at http://127.0.0.1:5401/admin/nodes
EVENT "{\"name\":\"InstanceStarted\",\"payload\":{\"instance_name\":\"sensei\",\"api_host\":\"127.0.0.1\",\"api_port\":5401,\"network\":\"bitcoin\",\"version\":\"0.2.1\",\"region\":null}}"
starting XXX@127.0.0.1:10000
```

I can login to admin and I have 1 node: w3n1

I can start and stop this node.

I can login to this node and it shows a left nav of: Chain, Channels, Send Money, Receive Money, Peer Directory, Logout

If I fund this node, where are the funds going to? A wallet in bitcoincore?
The address generated in the QR code is not in any list of addresses I have received to in either wallet in bitcoincore.

Then Open a channel?

Then?

thanks!

**Comment by johncantrell97** [comment]
Hi there,

The lightning node maintains it's own on-chain wallet.  The addresses it generates are sourced from this wallet and are used when opening and closing channels.  Yeah, once you have a funded node you can use it on the lightning network like you would lnd/cln/eclaire by opening channels, creating invoices, and paying invoices.

**Comment by w3evolutions** [comment]
Got it. I will need to read up on the lightning network again. It's been about 18 months :)

Thanks!

**Comment by w3evolutions** [comment]
Where do I get the:

Node Connection Info (pubkey@host:port)

To open a channel?

**Reply by johncantrell97** [reply]
You should be able to get it from the admin interface via the `Nodes` list page.  But yeah, it should be added to the actual node interface as well.

**Reply by w3evolutions** [reply]
0287d207a8aee4be70521470498e2203b24d98d80011568873a7ea346596eff3a8@127.0.0.1:10000
The CONNECTION INFO is for 127.0.0.1:1000, if I open a channel how will others connect? What port(s) needs to be open on the router / firewall?

thanks!
